lib/cucumber/core/test/result.rb

```diff
@@ -5,13 +5,19 @@ module Core
     module Test
       module Result
 
+        # Defines a status reader on a result class for the given status
+        #
         # Defines predicate methods on a result class with only the given one
         # returning true
-        def self.status_queries(status)
+        def self.status_methods(status)
```

Comment:
I think it's still OK to call it queries - these are all query rather than command methods.